serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1772,14 +1773,50 @@ export class _Exporter {
         });
     }
 
+    /**
+     * Getting the nodes and materials that would be exported.
+     * @param nodes Babylon transform nodes
+     * @returns Array of nodes which would be exported.
+     * @returns Set of materials which would be exported.
+     */
+    private getExportNodes(nodes: Node[]): [Node[], Set<Material>] {

```

Comment:
Super minor, but private functions should start with underscore. We can fix this later.